test/language/object-literal/object-literal-let-non-strict.js

```diff
@@ -0,0 +1,12 @@
+// Copyright (C) Copyright 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 12.2.5
+description: >
+    In non-strict mode, let is a valid Identifier.
+flags: [noStrict]
+---*/
+var let = 1;
+var object = {let};
```

Comment:
Or is it `property-definition-literal-property-name-let`? Sorry, I'm having a little trouble with the grammar